src/renderer/ipc.ts

```diff
@@ -13,7 +13,7 @@ function getImplementation(): IPCInterface {
 
   logger.log('IPC was not configured in preload script, falling back to custom protocol and fetch');
 
-  fetch(`${PROTOCOL_SCHEME}://${IPCChannel.PING}`).catch(() =>
+  fetch(`${PROTOCOL_SCHEME}://${IPCChannel.PING}/sentry_key`).catch(() =>
```

Comment:
Shouldn’t this be a query string and not part of the path?

Reply:
It doesn't matter since the `fetch` breadcrumb checking [matches the entire url](https://cs.github.com/getsentry/sentry-javascript/blob/d64550effb035d069a956e1fdc1b45ba51d289fc/packages/browser/src/integrations/breadcrumbs.ts#L207) and in the main process, the protocol handler returns an empty response for anything that isn't `scope` or `event`.

Reply:
We already use this to exclude `event` and `scope` messages but it was left off `ping`!

https://github.com/getsentry/sentry-electron/blob/ffad61d621119eee8f6d42e361e393fcdb478905/src/renderer/ipc.ts#L27

Reply:
Ahhhh I see, thanks for clarifying.